src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,17 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Checks the information stored is private or not in the address book.
+ */
+
+public class Contact {
+	 private boolean isPrivate;
```

Comment:
The way you have generalised the methods is not good. Try to identify more common methods which you can implement in the super class.